cql2pgjson/src/main/java/org/z3950/zing/cql/cql2pgjson/CQL2PgJSON.java

```diff
@@ -573,87 +566,53 @@ private String pg(CQLBooleanNode node) throws QueryValidationException {
   }
 
   /**
-   * Return SQL regexp expressions for do an "all" CQL match of the cql string.
-   * "all" means that all words match, in any position.
-   * <p>
-   * It matches if all returned expressions are true, the caller needs to "AND" them.
-   * Words are delimited by whitespace, punctuation, or start or end of field.
+   * Return SQL full text search expression for an "Adj/And/Any" CQL match string.
    *
    * @param textIndex  JSONB field to match against
    * @param modifiers  CqlModifiers to use
-   * @param cql   words to convert
-   * @return resulting regexps
+   * @param cql  words to convert
+   * @param ftOperator full text search operator to use. One of "<->", "&", or "|"
+   *
+   * @return resulting full text search expression
+   * @throws QueryValidationException
    */
-  @SuppressWarnings("squid:S1192")  // suppress "String literals should not be duplicated"
-  private static String [] allRegexp(String textIndex, CqlModifiers modifiers, String cql) {
+  private static String [] pgFtAAA(String textIndex, CqlModifiers modifiers, String cql, String ftOperator) throws QueryValidationException {
```

Comment:
The name `pgFtAAA` does not help understanding the source code. Can you rename to `fulltext` or some other better name?
`cql` should be renamed to `cqlTerm` because it is not a full CQL query.

Reply:
This method won't be needed after CQLPG-81 is done.